### PR TITLE
TRLL-01: Fixed icon behaviour in datatable

### PR DESCRIPTION
**Pull request by sanderPostma**
This was branched off triall while there is also a triall-merge branch. Not sure if triall-merge was used in the dployment. (I think was saw that an earlier image is in use by Triall)

### CODE REVIEW

File: lib/core/datatable/components/datatable/datatable.component.ts [hunk 1]
```diff
@@ -526,7 +526,8 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
     asIconValue(row: DataRow, col: DataColumn): string {
         if (this.isIconValue(row, col)) {
             let value = row.getValue(col.key) || '';
-            return value.replace('material-icons://', '');
+            let result = value.replace('material-icons://', '');
```
Comment (sanderPostma) [inline lib/core/datatable/components/datatable/datatable.component.ts]: Was for debug, should be reverted

Comment (LDMGN) [reply lib/core/datatable/components/datatable/datatable.component.ts]: Please revert in that case.

File: lib/core/datatable/components/datatable/datatable.component.html [hunk 2]
```diff
@@ -96,7 +96,7 @@
                                 </mat-icon>
                                 <ng-template #no_iconvalue>
                                     <mat-icon class="adf-datatable-selected"
-                                              *ngIf="row.isSelected; else no_selected_row" svgIcon="selected">
+                                              *ngIf="row.isSelected && col.key=='$thumbnail'; else no_selected_row" svgIcon="selected">
```
Comment (LDMGN) [inline lib/core/datatable/components/datatable/datatable.component.html:99]: What is the functionality that is achieved with this change?

Comment (sanderPostma) [reply lib/core/datatable/components/datatable/datatable.component.html:99]: The change is that only columns with '$thumbnail' as key (the first one) is replaced with the selected icon, not all mat-icon columns.

File: lib/core/datatable/components/datatable/datatable.component.ts (hunk 1 above)
Comment (sanderPostma) [reply lib/core/datatable/components/datatable/datatable.component.ts]: done

File: lib/core/datatable/components/datatable/datatable.component.ts [hunk 3]
```diff
@@ -505,12 +505,12 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
...
     onImageLoadingError(event: Event, row: DataRow) {
         if (event) {
-            let element = <any> event.target;
+            let element = <any>event.target;
 
             if (this.fallbackThumbnail) {
                 element.src = this.fallbackThumbnail;
             } else {
-                element.src = row.imageErrorResolver(event);
+                element.src = '//:0';
```
Comment (sanderPostma) [inline lib/core/datatable/components/datatable/datatable.component.ts:513]: The imageErrorResolver duplicates the icon of the first column into the blockchain state column. We just want a blank image when the state is not set.

**Review by LDMGN — APPROVED** [review]

**Pull request merged**